lib/sanbase/billing/billing.ex

```diff
@@ -70,4 +72,27 @@ defmodule Sanbase.Billing do
       {:error, _} -> true
     end)
   end
+
+  @spec create_or_update_stripe_customer(%User{}, String.t() | nil) ::
+          {:ok, %User{}} | {:error, %Stripe.Error{}}
+  def create_or_update_stripe_customer(_, _card_token \\ nil)
```

Comment:
Here you do not need to underscore vars. Let's keep the proper names for better documentation:
```suggestion
  def create_or_update_stripe_customer(user, card_token \\ nil)
```